row_variation_rankings.md

```diff
@@ -52,7 +52,7 @@ Unilateral barbell row with one end anchored, while placing the other hand and k
 
 **Fact-Check Result:** Personal Opinion/Misleading
 
-**Explanation:**
+**Explanation:**  The influencer said that BB bend over is the worst exercise, but the main thing is on what basis he made this claim. The barbell row exercise is one of a number of rowing exercises used to enhance strength and muscle development in the upper back, posterior shoulder girdle, and shoulder joint muscles, respectively. It often is taught relatively early in a strength and conditioning program because of its relative simplicity. Its utility as a safe and effective strength development tool is predicated on sound instruction, effective supervision, and proper execution. We should also keep in mind that BB bend over also helps in developing our stability as well as core strength (training the transverse abdominis isometrically). So in this case, personal bias comes from the influencer's end.
```

Comment:
I would go one by one - 1) The influencer made the claims on BB bent over row based on Mechanical tension profile(but don't have any any evidence) as stabilization becomes an issue when Mechanical tension is one's priority. 2) If only Mechanical Tension is the priority then, isolation is better. 3) BB bent over row is not a simple exercise, it might be taught early in the syllabus of SnC because we have more complex movements for training in SnC e.g. clean or hang clean. So, when someone says BB bent over row is simple - Is it simple for General population or Athletes?  
4) Focus on core strength, I have some questions - Do people need to train core with BB bent over row? Since, Core is being engaged in almost every standing free weight exercise(intensity differs in different exercise)- why to choose BB bent over row over other? SO a more big question is - What is Core? Which muscles to we include in Core and which not?....
But But But...
Let's look at the other side...
1) Influencer made the claim based on Mechanical tension profile for the sole purpose of Hypertrophy, but we still don't know the ACTUAL EXACT stimulus of Hypertrophy( Yes, Mechanical Tension has promising results but at what load and Metabolic stress still stands. Though, Muscle damage was considered earlier but it faded away, that's a different topic).  So we still don't have that "FULL PROOF PAKKA EVIDENCE" for Mechanical tension because we don't know which sensor actually initiates the mTORC in body to kick Protein Synthesis. I would repeat - Mechanical tension has promising results but not "FULL PROOF PAKKA" becasue opposition and loopholes exists. (BTW, the stimulus of Hypertrophy and the sensor is still a BIG question)

2)  Now, Applicability - Why to isolate the muscle when we actually know that Pulling action is a fundamental part of Human movement? Train movements not muscles is better explained by Abhinav Sir, toh yhn zyaada likhne ki zaroorat  nhi h.

3) Strength and Conditioning (one of my fav. apart from Biology and motor control) - So, BB bent over is taught as it is a specific exercise for some sports to train movements with resistance to make the movement Stronger and Powerful(focus Stronger annnnddd Powerful). It's transferability is higher for sports like Rowing. And, athletes don't have time to train muscles in isolation(2-3 sessions a day) they will always prioritize movement. 

My final note - The influencer gave information only to attract people who want to build a bigger back and backed with Mechanical tension profiles. But, the overall information about Hypertrophy is still in Question despite promising results. Scientists say "We can't say it clearly". Since, the video is uploaded on SC, people should be taught that yes, Hypertrophy is not everything. People with different needs and backgrounds will watch the video(It's kind strange that a 30-40 minute video can work with everybody). Overall, he might be giving the right information but not full information.

Reply:
And Yes, People might also say that - Muscles don't know in what way they are moving the body, they are just contracting and lengthening - but our neurons do know!!

Reply:
"I have some questions - Do people need to train core with BB bent over row?" 
Here i am just discuss that core is engaged while doing BB bend over, and I am not said specifically that you should train your core through BB bend over , core act as an Stabilizer so here I am add in the explanation.